examples/js/controls/FirstPersonControls.js

```diff
@@ -149,45 +149,45 @@ THREE.FirstPersonControls = function ( object, domElement ) {
 
 		//event.preventDefault();
 
-		switch ( event.keyCode ) {
+		switch ( event.code ) {
 
-			case 38: /*up*/
-			case 87: /*W*/ this.moveForward = true; break;
+			case 'ArrowUp': /*up*/
+			case 'KeyW': /*W*/ this.moveForward = true; break;
```

Comment:
minor: could remove the inline comments since `event.code` values are more readable.

Reply:
ok I'm doing this.
So should I push a new commit or modify the existing one and force-push ?
Pushing updates after changes were approved seemed to be an issue... Sorry for the confusion.

Reply:
> Pushing new commits after changes were approved seemed to be a problem

I would say changing code after a review is problematic no matter if you add new commits or force-push. 

In this particular case (where you just remove obsolete comments), you can do as you like.

Reply:
PS: You might also want to remove these comments in #21055.

Reply:
@Mugen87 ok, I've made the changes with a new commit in each PR